src/components/CombinedFeaturePanel/editors/AutoEditor.tsx

```diff
@@ -59,7 +58,7 @@ export const AutoEditor = ({ feature, tagKey }: AutoEditorProps) => {
   const { tagName, osmType, osmId } =
     useRetrieveOsmParametersFromFeature(osmFeature);
 
-  const [newTagValue, setEditedTagValue] = useState<EditorTagValue>("");
```

Comment:
I've removed this for now as the **generic** editor needs no knowledge about which kind of tag values are allowed. This is knowledge that belongs in its child implementations.